src/category_theory/monad/limits.lean

```diff
@@ -24,13 +24,13 @@ include 𝒥
 namespace forget_creates_limits
 variables (D : J ⥤ algebra T) [has_limit.{v₁} (D ⋙ forget T)]
 
-@[simps] def γ : (D ⋙ forget T ⋙ T) ⟶ (D ⋙ forget T) := { app := λ j, (D.obj j).a }
+@[simps, nolint doc_blame] def γ : (D ⋙ forget T ⋙ T) ⟶ (D ⋙ forget T) := { app := λ j, (D.obj j).a }
 
-@[simps] def c : cone (D ⋙ forget T) :=
+@[simps, nolint doc_blame] def c : cone (D ⋙ forget T) :=
```

Comment:
I think that's exactly the kind of definition that should have a docstring.  The name is completely unhelpful, and there seems to be more than one reasonable definition with this type.

You don't have to add a docstring here if you aren't motivated.  The definition is "grandfathered" in, which means it was added before we required documentation for everything and this exception is listed in the `scripts/nolints.txt` file.

Reply:
I've avoided adding a docstring here, but I've added them for the new proofs